static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -287,11 +303,47 @@ function ReleaseContext(props: ReleaseContextProps) {
   }
 
   return (
-    <Fragment>
+    <Wrapper data-test-id="quick-context-hover-body">
       {renderReleaseDetails()}
       {renderIssueCountAndAuthors()}
       {renderLastCommit()}
-    </Fragment>
+    </Wrapper>
+  );
+}
+
+function EventContext(props: BaseContextProps) {
+  let stackTrace: StacktraceType | null = null;
+  const {isLoading, isError, data} = useQuery<Event>(
+    [
+      `/organizations/${props.organization.slug}/events/${props.dataRow['project.name']}:${props.dataRow.id}/`,
+    ],
+    undefined,
```

Comment:
No need to pass `undefined` here, the options object should be able to be used as the second argument

Reply:
Hey @malwilley, I passed undefined there mainly because queryclient doesn't seem to pass down queryoptions as the third argument here [link](https://github.com/getsentry/sentry/blob/a5fbd4928ed0fe79c1ad14adb401507160819957/static/app/utils/queryClient.tsx#L56), if we pass the options object as the second param.

Reply:
Good catch! Keep this as-is then and I'll go ahead and fix that in queryClient

Reply:
Sounds good!